### ISSUE TITLE
fitPanelWidth: true without changing height

### ISSUE THREAD

**Issue by sSalvaXx** [pyrochlore/obsidian-tracker#190]
Is it possible to use the full available width of the panel without the output taking also the available height ? In other words, can I expand a line or bar graph horizontally without changing its vertical size ? Thanks in advance

**Comment by nikbrunner** [comment]
I also would love this feature

**Comment by codeRSH** [comment]
Yes, I am also looking for this feature. Please implement. Thanks for the great plugin.
